data-prepper-plugins/parse-json-processor/src/main/java/com/amazon/dataprepper/plugins/processor/parsejson/ParseJsonProcessor.java

```diff
@@ -76,9 +88,62 @@ public void shutdown() {
 
     }
 
+    private Map<String, Object> parseUsingPointer(final Event event, final Map<String, Object> parsedJson, final String pointer,
+                                                  final boolean doWriteToRoot) {
+        final Event temporaryEvent = JacksonEvent.builder().withEventType("event").build();
```

Comment:
We might want to try to avoid creating a temporary Event and find another solution, but this can come later.